src/api/displayTypes.ts

```diff
@@ -132,6 +132,10 @@ export interface Styles {
   edge: Array<IStyleRule<EdgeStyle>>;
 }
 
+export interface DefaultStyles extends Styles {
+  edgeGroup: EdgeStyle;
+}
+
```

Comment:
Why not add `edgeGroup` directly to the `Styles` interface ?

Reply:
Actually I understand this know, after reviewing the backend code.
Could you rename it to `DataSourceDefaultStyles` though to be consistent with the backend code
or any other name, as long as it is used and understood consistently everywhere